data/Dockerfiles/acme/docker-entrypoint.sh

```diff
@@ -18,18 +18,14 @@ while true; do
 		-f ${ACME_BASE}/acme/private/account.key \
 		-k ${ACME_BASE}/acme/private/privkey.pem \
 		-c ${ACME_BASE}/acme \
-		$*
+		${MAILCOW_HOSTNAME} autoconfig.${MAILCOW_HOSTNAME#*.} autoconfig.${MAILCOW_HOSTNAME#*.} ${ADDITIONAL_SAN}
 
 	case "$?" in
 		0) # new certs
 			# cp the new certificates and keys 
 			cp ${ACME_BASE}/acme/fullchain.pem ${ACME_BASE}/cert.pem
 			cp ${ACME_BASE}/acme/private/privkey.pem ${ACME_BASE}/key.pem
 
-			# also generate new dhparams with new certs
-			#(on the first run, the old dhparams may be the fakeoil from git)
-			openssl dhparam -out ${ACME_BASE}/dhparams.pem 2048
```

Comment:
But... why does dovecot regenerate `dhparams.pem` while it has snakeoil certs? So if dhparams get regenerated, but not the certs, ssl will still be broken (dovecot, nginx **and postfix**). What's the point on regenerating dhparams without knowing if SSL termination will be successful?

I can hear mailbox.org loudly crying about the majority of mailservers having only broken SSL.